dds/DCPS/Logging.cpp

```diff
@@ -18,15 +18,13 @@ namespace DCPS {
 void OpenDDS_Dcps_Export log_progress(const char* activity,
                                       const GUID_t& local,
                                       const GUID_t& remote,
-                                      const TimeDuration& duration,
-                                      size_t messages_sent,
-                                      size_t messages_received)
+                                      const MonotonicTime_t& start_time,
+                                      const GUID_t& reference)
 {
-  ACE_DEBUG((LM_INFO, "(%P|%t) {transport_debug.log_progress} local: %C remote: %C time(ms): %Lu msg_sent: %B msg_recv: %B activity: %C\n",
-             DCPS::LogGuid(local).c_str(), DCPS::LogGuid(remote).c_str(),
-             duration.value().msec(),
-             messages_sent, messages_received, activity));
-
+  ACE_DEBUG((LM_INFO, "(%P|%t) {transport_debug.log_progress} local: %C remote: %C reference: %C time(ms): %Lu activity: %C\n",
+             DCPS::LogGuid(local).c_str(), DCPS::LogGuid(remote).c_str(), DCPS::LogGuid(reference).c_str(),
```

Comment:
Since `reference` is optional, should this use a different format string when it's `GUID_UNKNOWN`?

Reply:
It makes the logs easier to parse if they are uniform.